lib/deface/override.rb

```diff
@@ -196,7 +196,7 @@ def digest
     #
     def self.digest(details)
       overrides = self.find(details)
-      to_hash = overrides.inject('') { |digest, override| digest << override.digest }
+      to_hash = overrides.inject(''.freeze) { |digest, override| digest << override.digest }
```

Comment:
This needs to become a `+=`.

Reply:
Do you mean `overrides.inject('') { |digest, override| digest += override.digest }` ?

Uhm maybe also `overrides.map(&:digest).join`?

Reply:
@ProGM I mean:
```ruby
overrides.inject(''.freeze) { |digest, override| digest += override.digest }
```
`<<` will fail on a frozen string because it attempts to mutate the object, so you need to create a new string every time. I'm not sure if that will negate the performance benefits of using `.freeze` though.